taxi/tests/test_forms.py

```diff
@@ -0,0 +1,19 @@
+from django.test import TestCase
+
+
+from taxi.forms import DriverCreationForm
+
+
+class FormsTest(TestCase):
+    def test_driver_creation_form_with_license_first_last_name_is_valid(self):
+        form_data = {
+            "username": "test",
+            "password1": "1A2g3s4f1234",
+            "password2": "1A2g3s4f1234",
+            "first_name": "test",
+            "last_name": "test",
+            "license_number": "ABC12345",
+        }
+        form = DriverCreationForm(data=form_data)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
```

Comment:
The assertion `self.assertEqual(form.cleaned_data, form_data)` may not work as expected because `form.cleaned_data` will not include the password fields (`password1` and `password2`). You might want to compare only the fields that are actually cleaned and available in `form.cleaned_data`.